_episodes/09-conflict.md

```diff
@@ -16,8 +16,8 @@ As soon as people can work in parallel, they'll likely step on each other's
 toes.  This will even happen with a single person: if we are working on
 a piece of software on both our laptop and a server in the lab, we could make
 different changes to each copy.  Version control helps us manage these
-[conflicts]({{ page.root }}/reference/#conflicts) by giving us tools to
-[resolve]({{ page.root }}/reference/#resolve) overlapping changes.
+[conflicts]({{ page.root }}/reference#conflicts) by giving us tools to
```

Comment:
It looks like our lesson has an additional error on this line, where the link `[conflicts]({{ page.root }}/reference#conflicts)` should actually be `[conflicts]({{ page.root }}/reference#conflict)` (because the definition on the page isn't pluralized).